File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.cs
public class Solution {
    public int MaximumCount(int[] nums) {
        int i, negCount = 0; 
        for (i = 0; i < nums.Length; i++) {
            if (nums[i] < 0) {
                negCount++;
            }
            if (nums[i] > 0) {
                break;
            }
        }
        return Math.Max(negCount, (nums.Length - i));
    }
}

File: 2614-maximum-count-of-positive-integer-and-negative-integer/maximum-count-of-positive-integer-and-negative-integer.py
class Solution:
    def maximumCount(self, nums: List[int]) -> int:
        neg_count = 0
        i = 0

        while i < len(nums):
            if nums[i] < 0:
                neg_count += 1
            if nums[i] > 0:
                break
            i += 1

        return max(neg_count, len(nums) - i)